storage/gr716-flash/flashsrv.c

```diff
@@ -387,6 +387,16 @@ static void flashsrv_rawThread(void *arg)
 				mutexUnlock(memory->lock);
 				break;
 
+			case mtGetAttr:
```

Comment:
I'd add raw IO check (applies also for `imxrt-flash`)

```suggestion
			case mtGetAttr:
				if (flashsrv_verifyRawIO(memory, &msg, &msg.o.size) < 0) {
					msg.o.err = -EINVAL;
					break;
				}
```

Reply:
I moved id check to the common code